apps/laboratory/tests/shared/fixtures/w3m-email-fixture.ts

```diff
@@ -47,7 +47,9 @@ export const testMEmail = base.extend<ModalFixture>({
         otp = email.getOtpCodeFromBody(emailBody)
       }
     }
-    if (otp.length !== 6) {
+    if (otp.length !== 7) {
```

Comment:
Is this ok?

Reply:
Yep. Otp is in format `123 456`. Empty space is counted as a char